### ISSUE TITLE
Suggestion: Add descriptions of list items

### ISSUE THREAD

**Issue by golfinq** [DJLink/list-of-game-engines-and-frameworks#11]
While this is very useful, adding a place (it could be in seperate files for easier PRs) for short description of each list item would improve the "what does using this mean from a game-dev perspective" aspect of the list

Edit - a resource which might help: https://enginesdatabase.com/

**Comment by DJLink** [comment]
The main idea was just a place where people could at least know these exist, but you're right, it's a lot of content with no info.  And that could be interesting, especially as it grows longer, but separate files that one would click on the main list we might as well send the user to the official webpage where each creator probably describes the project better then we can. putting them in the main list could get crowded quickly if we don't keep descriptions short. 🤔 I have to think about this.

I guess one thing that could help a lot would be programming languages supported or platforms etc? 


Edit: Just saw the link you edited with, yeah that does look nice, I guess it does work in that nice layout. cool thanks for sharing

**Comment by golfinq** [comment]
> The main idea was just a place where people could at least know these exist

Yeah engines and frameworks definitely should be known more widely - I follow Gamefromscratch and even they miss stuff

> official webpage where each creator probably describes the project better then we can. putting them in the main list could get crowded quickly if we don't keep descriptions short. 🤔

Agreed

> I guess one thing that could help a lot would be programming languages supported or platforms etc?

Yeah that is what I was thinking
